src/OpenConext/Profile/Value/Url.php

```diff
@@ -33,7 +33,7 @@
     public function __construct($url)
     {
         if (filter_var($url, FILTER_VALIDATE_URL) === false) {
-            throw new InvalidArgumentException('URL must be valid');
+            throw new InvalidArgumentException('URL must be valid: "' . $url . '"');
```

Comment:
Just personal preference, but I always prefer the use of `sprintf` in these situations.

`throw new InvalidArgumentException(sprintf('URL must be valid: ''%s"', $url));`

But don't sweat it!